apps.yaml

```diff
@@ -130,3 +130,15 @@ tb-mfh:
         description: App to run mean-field Hubbard tight-binding calculations.
         title: Empa nanotech@surfaces Laboratory - MFH TB for nanographenes
         state: development
+sssp:
+  releases:
+    - url: 'git+https://github.com/aiidalab/aiidalab-sssp@main:'
+      metadata:
+        categories:
+        - quantum
+        logo: https://raw.githubusercontent.com/aiidalab/aiidalab-sssp/main/miscellaneous/logo-sssp.png
+        authors: Jusong Yu, EPFL
+        description: AiiDAlab app to run pseudopotential verification.
+        title: SSSP verification
+        state: development
```

Comment:
It is no longer necessary to provide the metadata in the registry unless it is missing from the repository or one would want to explicitly override it – a complete `setup.cfg` file is sufficient. I've created https://github.com/aiidalab/aiidalab-sssp/pull/1 to add a bit more information and to match the metadata provided here. After it is merged, we can merge here.
```suggestion
```